app/_hub/kong-inc/acme/index.md

```diff
@@ -115,7 +115,23 @@ verify Let's Encrypt API. The CA-bundle file is usually `/etc/ssl/certs/ca-certi
 Ubuntu/Debian and `/etc/ssl/certs/ca-bundle.crt` for CentOS/Fedora/RHEL.
 
 #### Enable the Plugin
+
+For all the domains that you need to get certificate, make sure `DOMAIN/.well-known/acme-challenge`
+is mapped to a Route in Kong. You can check this by sending
+`curl KONG_IP/.well-known/acme-challenge/x -H "host:DOMAIN"` and expect a response `Not found`.
+If not, add a Route and a dummy Service to catch this route.
 ```bash
+# add a dummy service if needed
+$ curl http://localhost:8001/service \
+        -d name=acme-dummy \
+        -d url=http://127.0.0.1:65535
+# add a dummy route if needed
```

Comment:
Can you add a blank line right before this for consistency with the "add the plugin" section?

Reply:
Added, thank you!